tests/helpers/flash-message.js

```diff
@@ -0,0 +1,3 @@
+import FlashObject from 'ember-cli-flash/flash/object';
+
+FlashObject.reopen({ init() {} });
```

Comment:
Why?

Reply:
Again, why is this needed?

Reply:
I didn't added that it was created automatically when I installed ember-cli-flashmessages.